src/submission/exceptions/exceptions.ts

```diff
@@ -161,52 +189,54 @@ export const checkForProgramAndEntityExceptions = async ({
 
   // if there are submitted exceptions for this program, check if they match record values
   if (!(programException || entityException)) {
-    return { filteredErrors: schemaValidationErrors, normalizedRecord: record };
+    return { filteredErrors: schemaValidationErrors, normalizedRecord };
   }
 
   // check each validation error for a matching exception, and remove the validaiton if the value matches the exception value
   schemaValidationErrors.forEach(validationError => {
     const validationErrorFieldName = validationError.fieldName;
     const fieldValue = record[validationErrorFieldName];
```

Comment:
For numeric values with blank fields, `fieldValue` is `undefined` at this point. Will update so integer values are not 'normalized', and `undefined` is stored in the DB